packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -207,6 +240,10 @@ mixin SelectionRegistrant on Selectable {
       _subscribedToSelectionRegistrar = false;
     }
   }
+
+  // not applicable since we don't know the type here
```

Comment:
I think we can avoid having this if the `TextSelection` is self-contained within `SelectedContent`.